.github/workflows/aks_destroy_review.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   delete-review-app:
     name: Delete Review App ${{ github.event.pull_request.number }}
     concurrency: deploy_review_${{ github.event.pull_request.number }}
-    if: ${{ contains(github.event.pull_request.labels.*.name, 'review-app') }}
+    if: github.actor != 'dependabot[bot]' && github.event_name == 'pull_request'
```

Comment:
I think we could probably use `${{ !contains(github.event.pull_request.labels.*.name, 'dependencies') }}` (like the old workflows) as we're merging the prs so the actor will not be dependabot

Reply:
Have updated accordingly. Feels a bit weird not detecting the originating user directly but there's no obvious clean way of doing that.

`github.event.pull_request.user.login` seemed like it might.